pyproject.toml

```diff
@@ -79,6 +79,9 @@ git-cz = "commitizen.cli:main"
 cz_conventional_commits = "commitizen.cz.conventional_commits:ConventionalCommitsCz"
 cz_jira = "commitizen.cz.jira:JiraSmartCz"
 cz_customize = "commitizen.cz.customize:CustomizeCommitsCz"
+conventional_commits_asciidoc = "commitizen.cz.conventional_commits:ConventionalCommitsAsciiDoc"
+conventional_commits_textile = "commitizen.cz.conventional_commits:ConventionalCommitsTextile"
+conventional_commits_restructuredtext = "commitizen.cz.conventional_commits:ConventionalCommitsRst"
```

Comment:
This I was thinking that it could be it's own category, separated from the "rules" provider. The way we have in the new line `86` the "customer version providers". We could have `tool.poetry.plugins."commitizen.changelog"`.